sample/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'com.polidea.cockpit'
 
 buildscript {
+    ext.cockpit_core_version = '1.1.1-SNAPSHOT'
```

Comment:
What about extracting `cockpit_core_version` to `dependecies.gradle`?
`ext {
    ext.cockpit_core_version = '1.1.1-SNAPSHOT'
}`